network/p2p/inspector/validation/control_message_validation_inspector_test.go

```diff
@@ -351,12 +354,66 @@ func TestControlMessageValidationInspector_processInspectRPCReq(t *testing.T) {
 		})
 		require.NoError(t, inspector.processInspectRPCReq(inspectMsgReq))
 	})
+
+	t.Run("inspectRpcPublishMessages should disseminate invalid control message notification when invalid pubsub messages count greater than configured RPCMessageErrorThreshold", func(t *testing.T) {
+		inspector, distributor, _, subscriptions, _, sporkID := inspectorFixture(t)
+		// 5 invalid pubsub messages will force notification dissemination
+		inspector.config.RPCMessageErrorThreshold = 4
+		// create unknown topic
+		unknownTopic := channels.Topic(fmt.Sprintf("%s/%s", unittest.IdentifierFixture(), sporkID)).String()
+		// create malformed topic
+		malformedTopic := channels.Topic("!@#$%^&**((").String()
```

Comment:
```suggestion
		malformedTopic := unittest.RandomStringFixture(t, 100)
```
Please revise the malformed topic instances to use [the `unittest.RandomStringFixture`](https://github.com/onflow/flow-go/blob/91da6d0b24d1d5a5e156c3616f1179b4951d19a6/utils/unittest/strings.go#L12) instead of manually creating a malformed topic.

Reply:
https://github.com/onflow/flow-go/pull/4720/commits/7e21e5e8f015f7748acac37d3933a1a448f1e972